hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.cc

```diff
@@ -34,4 +34,17 @@ void HologramDataSourceAvailabilityFetcher::
                                                 &hologram_configs_));
 }
 
+std::string HologramDataSourceAvailabilityFetcher::ParseFlags() {
+    // Ensure that all the flags are given.
+    assert(absl::GetFlag(FLAGS_chipper_batch_job_cell) != "");
```

Comment:
!absl::GetFlag(FLAGS_chipper_batch_job_cell).empty()

Reply:
Can you show me where's the api for this function call? I can't seem to find one

Reply:
Sorry I thought you were referring a special function that checks if a flag is empty, I have changed it accordingly